core/src/test/java/com/xmartlabs/base/core/DateHelperTest.java

```diff
@@ -46,6 +56,139 @@ public void getListOfDaysBetweenTwoWeeks() {
 
     List<LocalDate> listOfDaysBetweenTwoDates = DateHelper.getListOfDaysBetweenTwoDates(firstDayOfWeek, end);
 
-    Assert.assertEquals(expectedList, listOfDaysBetweenTwoDates);
+    assertThat(expectedList, equalTo(listOfDaysBetweenTwoDates));
+  }
+
+  @Test
+  public void getFirstDayOfWeekMonday() {
+    LocalDate expectedMonday = LocalDate.of(2017, Month.APRIL, 10);
```

Comment:
Can this parameters be constants? For example, the month and the year are repeated in almost all the test methods.